sys/posix/sockets/posix_sockets.c

```diff
@@ -211,6 +211,8 @@ static int _ep_to_sockaddr(const struct _sock_tl_ep *ep,
 static int _sockaddr_to_ep(const struct sockaddr *address, socklen_t address_len,
                            struct _sock_tl_ep *out)
 {
+    assert(address != NULL);
```

Comment:
I ran our cppcheck script and that printed a warning (not an error) for this. Apart from this, the other change regarding (`recv_timeout`) was reported by murdock, cause for static test no sock module (IP, TCP, nor UDP) is active.